qds_sdk/commands.py

```diff
@@ -1417,6 +1418,16 @@ def parse(cls, args):
                 options.macros = validate_json_input(options.macros, 'Macros', cls)
             if options.retry is not None:
                 options.retry = int(options.retry)
+            if options.upload_to_source is not None:
+                options.upload_to_source = options.upload_to_source.lower()
+                if options.upload_to_source == 'true':
+                    options.upload_to_source = True
+                elif options.upload_to_source == 'false':
+                    options.upload_to_source = False
+                else:
+                    msg = 'Upload to Source parameter takes value of either True \
+                    or False only.'
```

Comment:
```suggestion
                    msg = "Upload to Source parameter takes a value of either 'true' \
                    or 'false' only."
```

Reply:
otherwise, the message gives the impression that the user needs to pass `True` in python-format